downstream/assemblies/aap-on-azure/assembly-aap-azure-network-peering.adoc

```diff
@@ -0,0 +1,41 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-aap-azure-network-peering"]
+= Private network peering
+
+:context: aap-azure-install
+
+// [role="_abstract"]
+// You can use these instructions to install 
+
+Ansible Automation Platform on Azure is deployed into an independent managed resource group with its own Azure virtual network (VNET). 
+
+When initially deployed, Ansible Automation Platform on Azure’s VNET can only send requests to external networks through the public internet.
+
+You can configure your Azure VNETs to allow private communication between multiple Azure VNETs as well as private transit routing between Azure VNETs and external VPN routed networks. These VPN networks can be on-premises or on other clouds.
+
+[NOTE]
+====
+While every effort has been made to align with Microsoft’s documentation for this content, there may be drift in accuracy over time.
+link:https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview[Microsoft’s documentation] is the source of truth for information about networking topics for Azure.
+====
```

Comment:
I like this note 👍🏽

Reply:
Yes, Red Hat doesn't want to support Microsoft's processes !  That's why there are links to Azure docs after every procedure :)